src/Controller/CRUDController.php

```diff
@@ -1677,6 +1683,31 @@ private function checkParentChildAssociation(Request $request, object $object):
         }
     }
 
+    /**
+     * Checks whether $needle is equal to $haystack or part of it.
+     *
+     * @param object          $needle   Object to compare with $haystack
+     * @param object|iterable $haystack Object to compare with $needle
+     *
+     * @return bool true when $haystack equals $needle or $haystack is iterable and contains $needle
+     */
+    private function equalsOrInList(object $needle, $haystack): bool
```

Comment:
```suggestion
    private function equalsOrContains(object $needle, $haystack): bool
```

Reply:
Is it clear that it means that param2 contains param1 ?

Reply:
I agree about the argument swapping.

Reply:
I made the changes